README.md

```diff
@@ -1,41 +1,33 @@
-
-## Optional Configs
-The following optional config parameters are available for the Odrive:
-1. `odrive_config_file`: path to a json file with your odrive configs (string)
-    * You can add this config parameter if you'd like to reconfigure your Odrive each time it is initialized, but if you have configured your Odrive to your liking already, it's not necessary.
-    * Extract your configurations from your odrive. To do so you must have `odrivetool` installed. See the [Odrive instructions](https://docs.odriverobotics.com/v/latest/odrivetool.html) on how to do this.
-    * After installing `odrivetool` run `odrivetool backup-config config.json` to extract your configs to a file called `config.json`. See [Odrive instructions](https://docs.odriverobotics.com/v/latest/odrivetool.html#configuration-backup) for more info.
-    * `iq_msg_rate_ms` in the config defaults to `0`, but it needs to be approximately `100` in order to use `set_power()` 
-    * Provide the path to the config file as a string for this parameter.
-    * See sample-configs/config.json for a sample odrive config file.
-    * (For `"canbus"` connection) If you add an `odrive_config_file`, you will have to leave the Odrive plugged in to the USB port in addition to wiring the CANH and CANL pins.
-    * An alternative to adding an `odrive_config_file` is running the command `odrivetool restore-config /path/to/config.json` in your terminal.
-2. `serial_number`: serial number of the odrive (string).
-    * This is not necessary if you only have one odrive connected. See *Troubleshooting* section *Hanging* for a note on multiple serial numbers. 
-    * (For `"canbus"` connection) This is not necessary unless you have multiple Odrives connected AND are providing an `odrive_config_file` for any of them. The `"canbus"` implementation allows you to connect multiple Odrives without providing a `serial_number` as long as you don't have any `odrive_config_file`.
-3. (Only with `"canbus"` connection) `canbus_baud_rate`: baud rate of the odrive CAN protocol (string).
-    * This parameter can be found using `odrivetool` with `<odrv>.can.config.baud_rate`
-    * Format the string as a multiple of 1000 (ex: `"250k"`)
+* See the [ODrive documentation](https://docs.odriverobotics.com/v/latest/getting-started.html) to configure and tune your motor properly. 
+This configuration remains on the same ODrive motor controller across reboots, and only changes when you go through the configuration of the ODrive again.
+* See the [ODrive CAN documentation](https://docs.odriverobotics.com/v/latest/can-guide.html) for detailed information on how to set up CAN on your ODrive. 
+Make sure that you have:
+    * enabled SPI communication on your SBC
+    * installed `odrivetool`, `python-can`, `cantools`, and a [`viam-sdk`](https://python.viam.dev)
```

Comment:
@randhid re. url, is this only supported in the Python SDK?

Reply:
yes, this is a python only module if that's what you're asking

Reply:
@randhid Ok cool -- I'll add a note indicating that for users into the docs page